### ISSUE TITLE
Did you use MAML like method to train your model?

### ISSUE THREAD

**Issue by lihuiliullh** [xwhan/One-shot-Relational-Learning#20]
Did you use MAML like method to train your model? If not, where is the meta part?

**Comment by zhangweijian97** [comment]
you can check MetaR https://aclanthology.org/D19-1431/

**Comment by lihuiliullh** [comment]
It seems like the authors didn't use it in the paper. I would like to confirm my guess.

**Comment by zhangweijian97** [comment]
you are right. This paper use metric-based meta learning. MAML belongs to gradient-based meta learning
